src/scss/components/_media-element.scss

```diff
@@ -0,0 +1,132 @@
+/**
+ * Media element
+ * ===================================
+ * Example – SERP – Restaurant logos
+ *
+ * A common pattern that sees an image followed by a block of information.
+ *
+ */
+
+$mediaElement-img-width: 50px;
+
```

Comment:
Super nit picky – any need for the spaces between these vars?

Reply:
It's a direct copy from INT CW, I can change it if you want but can we still aim to get this in today?